.syft.yaml

```diff
@@ -0,0 +1,6 @@
+# Only the Java service components are part of the final container image, so tests and demos should not be
+# part of any generated SBOM from Syft.
+exclude:
+  - ./.github
+  - ./explainability-service/demo/**
+  - ./tests/**
```

Comment:
Nit: consider adding trailing newline at the end of the file. It probably won't cause any issues, but I think it should be there (notice how GitHub highlights the absence of it here with a red marker).

More info:
https://unix.stackexchange.com/a/18789/86764
https://stackoverflow.com/a/729795/2829381